krm-functions/lib/condkptsdk/sdk.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2023 The Nephio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package condkptsdk
+
+import (
+	"fmt"
+
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+	kptfilelibv1 "github.com/nephio-project/nephio/krm-functions/lib/kptfile/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const FnRuntimeOwner = "fnruntime.nephio.org/owner"
```

Comment:
I propose to remove the "fnruntime" string from teh  "fnruntime.nephio.org/owner" annotation name, since the ownership relationship between kpt package resources is independent of what SDK we use for implementing the KRM functions. 
I.e. this annotation will be used also in Specializers and 3rd party KRM functions that might not use our SDK (might not even be implemented in Go).

Reply:
or specializer.nephio.org ?

Reply:
changed to specializer.nephio.org for now

Reply:
sgtm